gips/0053-enabling-substreams-based-subgraphs.md

```diff
@@ -0,0 +1,138 @@
+---
+GIP: <0053>
+Title: <Enabling-Substreams-Based-Subgraphs>
+Authors: <Alex Bourget alex@dfuse.io, Adam Fuller adam@edgeandnode.com>
+Created: <2022-07-19>
+Updated: <2023-06-21>
+Stage: <Draft>
+Discussions-To: <https://forum.thegraph.com/t/ggp-0025-enabling-substreams-based-subgraphs/4287, https://forum.thegraph.com/t/substreams-into-subgraphs-a-simple-integration/3542>
+---
+
+# Abstract
+
+Substreams are a new streaming-first blockchain indexing technology developed for The Graph Network, focused on performance and composability.
+
+A substreams package is a collection of interdependent modules, which combine to extract and transform blockchain events in a highly configurable and parallelisable way, producing a streaming output.
+
+This GIP describes the simplest possible integration of Substreams with Subgraphs, to take that final streaming output and make it persistent and queryable.
+
+# Motivation
+
+As Substreams become first-class citizens with The Graph Network, the StreamingFast team believes that Substreams software has been battle tested enough to be deemed fully production ready with The Graph Network. To ensure that Indexers will index substreams-back subgraphs, the GIP proposes to add Oracle support for dataSource.kind =="substreams" to the Feature Matrix shared below. 
```

Comment:
> As Substreams become first-class citizens with The Graph Network

This is not clear what this means

> the GIP proposes to add Oracle support for dataSource.kind =="substreams" to the Feature Matrix shared below.

I don't see a feature matrix in this document?